necst/ctrl/antenna/pid_controller.py

```diff
@@ -28,23 +28,31 @@ def __init__(self, **kwargs):
         self.create_subscription(
             PIDMsg, "pid_param", self.change_pid_param, qos.reliable
         )
-
         self.az = self.el = self.az_enc = self.el_enc = self.t = self.t_enc = None
+        self.list = []
+
+    def get_data(self, current):
+        sorted_list = sorted(self.list, key=lambda msg: msg.time)
+        for i in range(len(sorted_list)):
```

Comment:
色々言ってすみません、こちらの方がわかりやすいかもしれません。
```suggestion
        while len(sorted_list)>0:
```